binstar_client/commands/copy.py

```diff
@@ -13,10 +13,20 @@ def main(args):
 
     spec = args.spec
     channels = bs.list_channels(spec.user)
-    if args.from_channel not in channels:
-        raise errors.UserError("Channel %s does not exist\n\tplease choose from: %s" % (args.from_channel, ', '.join(channels)))
+    __name = 'label' if (args.from_label and args.to_label) else 'channel'
```

Comment:
I guess my point is that it's easy to confuse this with `__name__`. How about `label_text`? Or maybe just move the expression inline into the format statement:

```
"{label} {from_label} does not exist".format(
  label='Label' if (args.from_label and args.to_label) else 'Channel',
  from_label=from_label,
)
```